src/DataViews/DataView.cpp

```diff
@@ -95,6 +95,8 @@ void DataView::OnContextMenu(const std::string& action, int /*menu_index*/,
     OnEnableFrameTrackRequested(item_indices);
   } else if (action == kMenuActionDisableFrameTrack) {
     OnDisableFrameTrackRequested(item_indices);
+  } else if (action == kMenuActionIterate) {
+    OnIterateRequested(item_indices);
```

Comment:
Nit: I would prefer to call the action `kMenuActionAddIterator` and the function `OnIteratorRequested`

Reply:
Done! Thanks for the suggestion!